### ISSUE TITLE
Can't install any nix-gaming package when installing NixOS with `nixos-install`

### ISSUE THREAD

**Issue by NovaViper** [fufexan/nix-gaming#187]
Whenever I include any of the nix-gaming packages in my flake to install, such as in my gaming flag:

https://github.com/NovaViper/NixConfig/blob/6a369506a3d0b85601ec9d5179021a70f76e55b3/home/novaviper/features/games/default.nix#L13

I get the following error when I run `nixos-install --flake .#laptop --root /mnt`: https://pastebin.com/9UcFfnSe

**Comment by fufexan** [comment]
The error doesn't seem to stem from any particular package, when checking using `nix repl` and `:lf github:fufexan/nix-gaming`. Are you sure the source is actually the nix-gaming repo?

**Comment by NovaViper** [comment]
I'm not sure how I can check where exactly the issue is coming from. 🤷🏾‍♀️ I comment out the line for installing the package from the repo and `nixos-install` allows me to continue the installation; and I can install the package when I'm on an existing NixOS system.

**Comment by fufexan** [comment]
Yeah, I've had such problems with nixos-install before. Not sure what's up with it.

**Comment by Anomalocaridid** [comment]
I am also having this issue using `wine-ge`. I tested it replacing `wine-ge` with an arbitrary package (`modrinth-app`) and the problem does not seem to occur then.

Since @NovaViper is reporting this problem with `wine-discord-ipc-bridge`(according to the config they linked), could it somehow be an issue with the wine-related packages?

**Comment by fufexan** [comment]
Could be, yes.

**Comment by NobbZ** [comment]
I don't know what the exact error was, though there are various cases with `nixos-install --flake`, most of them related to either `/mnt/nix/store/…` leaking into the drvs build sandbox, or the build sandbox trying to access a path thats not in the target store, but in the store of the "host" system.

I have not been able to produce minimal examples, and do not know what the conditions are for this to hapen. Its just observations over the last years(?), mostly second hand.

All of these were "fixed" (better: worked around) by just generating a simple config, installing that, and using `nixos-rebuild boot --flake` and another reboot to apply the actual configuration.

I am not aware of any tracking issue at nixos/nix nor nixos/nixpkgs.
